dashboard/test/models/district_test.rb

```diff
@@ -4,9 +4,9 @@ class DistrictTest < ActiveSupport::TestCase
   test "destroying district removes it from a cohort" do
     cd = create :cohorts_district
     district = cd.district
-    district = district.reload
+    district.reload
     cohort = cd.cohort
-    cohort = cohort.reload
+    cohort.reload
```

Comment:
`.reload` already modifies the target (I'm not sure why it's not `.reload!` -- maybe because there's no non-`!` counterpart?)